augur/io/metadata.py

```diff
@@ -646,7 +646,7 @@ def _get_delimiter(path: str, valid_delimiters: Iterable[str]):
         if len(delimiter) != 1:
             raise AugurError(f"Delimiters must be single-character strings. {delimiter!r} does not satisfy that condition.")
 
-    with open_file(path) as file:
+    with open_file(path, newline='') as file:
```

Comment:
Should this be added everywhere a `csv` reader object is used? I looked through the codebase and there is one more:

https://github.com/nextstrain/augur/blob/2c882986bc9ed7cf683546ebab96250e07b9321d/augur/curate/apply_record_annotations.py#L32-L33

Reply:
Should probably also be added everywhere for `csv.writer`? 

https://github.com/nextstrain/augur/blob/2c882986bc9ed7cf683546ebab96250e07b9321d/augur/index.py#L49-L50

Reply:
Yeah. It should. I didn't because I didn't want to touch more bits of the codebase. But now [I've done so](https://github.com/nextstrain/augur/pull/1564)!